script/openqa-bootstrap

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash -x
+
+
+# add extra repos for leap
+source /etc/os-release
+if $NAME == "openSUSE Leap" ; then
+	zypper -n addrepo obs://devel:openQA devel:openQA
```

Comment:
I suggest the addrepo parameters "-f -p 105"

Reply:
`-f` is already the default.

Reply:
Does `-p 105` increase or decrease the priority?

Reply:
It decreases priority which should be fine for "devel:openQA" itself, meaning: Only if we can not find the package in the main repos we install packages from here. However, thinking about it, depending on the needed features we might want to prefer "devel:openQA" so `-p 95` would be more suitable

Reply:
--> https://github.com/os-autoinst/openQA/pull/1944